scripts/sign_encrypt.py

```diff
@@ -222,7 +227,6 @@ def assign_default_value(parsed, attr, func):
 
     return parsed
 
-
```

Comment:
That will make `pycodestyle` unhappy. CI hasn't detected this and many other coding style issues due to the checkpatch errors which make the CI script exit without even running `pycodestyle`. I need to see how to fix this. In the meantime, could you run `pycodestyle` on each commit and make sure any issues are addressed?